### ISSUE TITLE
Rasa X 0.33 with RASA 2.0.2 Docker compose setup issue on Windows 10 - Permission denied: '/.config'

### ISSUE THREAD

**Issue by amardeep2006** [RasaHQ/rasa#7190]
**Rasa version**: 2.0.2

**Rasa SDK version** (if used & relevant): na

**Rasa X version** (if used & relevant): 0.33

**Python version**: 3.7

**Operating system** (windows, osx, ...): Windows 10 - 1809 , Docker version 19.03.13, build 4484c46d9d

**Issue**:
I am not able to upload models/train model to rasa x UI. I see following error message on UI : Training failed . Your model training has failed.

**Error (including full traceback)**:
**Rasa x container error** 
```
Processed story blocks: 100%|██████████| 1/1 [00:00<00:00, 1822.82it/s, # trackers=1]ERROR:rasax.community.api.blueprints.models:Cannot connect to host rasa-worker:5005 ssl:default [Connection refused]

Encountered an exception while training. Please check the logs of the rasa worker container for more information.

```
**Errors in rasa-worker container** 

```
Starting Rasa X in production mode... 🚀
Could not fetch runtime config from server at 'http://rasa-x:5002/api/config?token=**********************=='. Exiting.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-hphg6cm2 because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.

```

**Command or request that led to error**:
```

```

**Content of configuration file (config.yml)** (if relevant):
```yml

```

**Content of domain file (domain.yml)** (if relevant):
```yml

```

**Comment by sara-tagger** [comment]
Thanks for the issue, @degiz will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by degiz** [comment]
Hey @amardeep2006 

Do you still experience the issue with Rasa Open Source `2.1.x` ?

**Comment by amardeep2006** [comment]
hi @degiz This issue is solved with RASA X 2.1.x and RASA X 0.33.1 onwards . 
I can confirm docker compose works with both 0.33.1 and 0.33.2 without problem.

**Comment by vaxin** [comment]
I encounter the same issue on docker image rasa/rasa:2.6.3-full
